docs/AI-for-security/api/anonymization-fields-api-find.asciidoc

```diff
@@ -0,0 +1,269 @@
+[[anonymization-fields-api-find]]
+=== Find AI Assistant anonymization fields
+
+Retrieve a list of Elastic AI anonymization fields.
```

Comment:
I'm not clear on what "anonymization fields" means here. It looks like the response includes fields that are and aren't currently set to be anonymized. "Anonymization fields" sounds like it's just the fields that are set to be anonymized. So what defines an anonymization field?